Scripts/Connector/Sample connector backup script.ps1

```diff
@@ -0,0 +1,42 @@
+# This script creates a backup for the connector in the machine in which it is run.
+# The files generated by this script will be saved in C:\ConnectorBackup, these files will
+# need to be copy to the machine in which you wish to restore the Connector.
+
+# RUN THIS SCRIPT IN AN ELEVATED POWERSHELL WINDOW
```

Comment:
is there a way to check if the Powershell is elevated? that will avoid failing part way through/partially failing